step-templates/iis-website-update-property.json

```diff
@@ -37,11 +41,11 @@
       }
     }
   ],
-  "LastModifiedOn": "2015-10-23T01:00:50.788+00:00",
-  "LastModifiedBy": "jmalczak",
   "$Meta": {
-    "ExportedAt": "2015-10-23T02:15:40.676Z",
-    "OctopusVersion": "2.6.5.1010",
+    "LastModifiedOn": "2016-05-30T11:42:50.429Z",
+    "LastModifiedBy": "ylashin78",

```

Comment:
Thanks Henrik,
I will do the updates.
By the way, many step templates like this one have a big try catch and do not fail the deployment if some error happens which is a bit misleading
Is this a common design pattern for them or it is the decision of the template author
RegardsYousry

Date: Mon, 30 May 2016 14:20:13 -0700
From: notifications@github.com
To: Library@noreply.github.com
CC: ylashin78@hotmail.com; author@noreply.github.com
Subject: Re: [OctopusDeploy/Library] Allow passing property names with any character case and also handle different types of web site properties in a better way (#329)

In step-templates/iis-website-update-property.json:

>    "$Meta": {
> -    "ExportedAt": "2015-10-23T02:15:40.676Z",
> -    "OctopusVersion": "2.6.5.1010",
> +    "LastModifiedOn": "2016-05-30T11:42:50.429Z",
> +    "LastModifiedBy": "ylashin78",

The above two lines should be top-level properties of the JSON, not a part of the $Meta property.


Also, I just noticed that the LastModifiedBy property doesn't match your GitHub username ylashin.


—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.